src/Features/LanguageServer/ProtocolUnitTests/Initialize/InitializeTests.cs

```diff
@@ -28,13 +28,13 @@ private static async Task<LSP.InitializeResult> RunInitializeAsync(Solution solu
 
         private static void AssertServerCapabilities(LSP.ServerCapabilities actual)
         {
-            Assert.True(actual.DefinitionProvider);
-            Assert.True(actual.ImplementationProvider);
-            Assert.True(actual.DocumentSymbolProvider);
-            Assert.True(actual.WorkspaceSymbolProvider);
-            Assert.True((bool)actual.DocumentFormattingProvider.Value);
-            Assert.True((bool)actual.DocumentRangeFormattingProvider.Value);
-            Assert.True(actual.DocumentHighlightProvider);
+            Assert.True((bool)actual.DefinitionProvider);
```

Comment:
Can we file a bug to have `SumType<T1, T2>.Value` be generic based on the value of `T1`? It returning object and us then needing to manually cast things feels horrible.